netcore/README.md

```diff
@@ -0,0 +1,5 @@
+1. install ASP.NET Core Runtime 3.1 as described [here](https://docs.microsoft.com/en-us/dotnet/core/install/linux-debian#install-the-runtime)
+1. clone repo
+1. navigate to /netcore
+1. make sure the **server.crt** is also present here
```

Comment:
I've just tried with my self signed certificate, created by openssl, and it failed with:
```
fail: Microsoft.AspNetCore.Server.Kestrel[0]
      Unhandled exception while processing 0HM3C3KJ4SVLQ.
System.NotSupportedException: The server mode SSL must use a certificate with the associated private key.
```

server.crt looks like:
```
-----BEGIN CERTIFICATE-----
MIIFqTCCA5ECFFMulPEMCLYhWZkGml/Gr7fg7Z5TMA0GCSqGSIb3DQEBCwUAMIGQ
MQswCQYDVQQGEwJCRzEQMA4GA1UECAwHVGFybm92bzEPMA0GA1UEBwwGVmVsaWtv
MQ8wDQYDVQQKDAZIdWF3ZWkxDDAKBgNVBAsMA09TUzEaMBgGA1UEAwwRdGVzdGJl
...
jts8bshmIBbpvIjqIA==
-----END CERTIFICATE-----
-----BEGIN RSA PRIVATE KEY-----
MIIJKQIBAAKCAgEAxD6/UgV5S2Au7B7LUcW8B8ykq4QTgzCX46lX8Lf0nSsM8v5b
vySY84Cc6OyHGmQk3d7wtG934/22H55XMdgout8tIjpiP7b3Rz1Z7ckeGpPp95qD
EwMoSTLQscJlNsfpf+gBwXrRiMwFG13s1HH+ArId7LtdYW3hPfjUjpAp03vE9zyj
...
QaDx4zb8A/9puKEJ9qlhVAWtDyByURu98UObUyRGB8YVMKH0XWULGNUmm24F
-----END RSA PRIVATE KEY-----
```